sci-astronomy/siril/siril-1.0.6-r3.ebuild

```diff
@@ -26,6 +26,7 @@ DEPEND="
 	>=dev-libs/json-glib-1.2.6
 	>=dev-libs/libconfig-1.4[cxx]
 	>=media-gfx/exiv2-0.25
+	<media-gfx/exiv2-0.28
```

Comment:
Unecessary. exiv2-0.28 remains masked and it should be possible to stabilise newer version of siril before exiv2-0.28 unmasking. Upper-bound version limits are bad UX and we're just letting stable users build new revision for no good reason.

Reply:
I understand. In this case we leave Siril 1.0.6 as it is and consider https://bugs.gentoo.org/906499 as resolved as it is already fixed upstream but not yet stable.